super-cool-game/Assets/Scripts/Player/PlayerStats.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerStats : MonoBehaviour {
+
+    public const int maxHealth = 100;
+    public int currentHealth;
```

Comment:
Bad idea to put this public.
If you need it public to be read, do `public int currentHealth { get; private set; }`
It should only be writable through appropriate methods